### ISSUE TITLE
Related issues

### ISSUE THREAD

**Issue by alexander-held** [alexander-held/agc-autodiff#3]
Gathering related issues to easily be able to track everything in one place.

- [ ] https://github.com/scikit-hep/awkward/issues/2591

**Comment by Saransh-cpp** [comment]
One year late, but can be closed now!

**Comment by alexander-held** [comment]
Indeed, thanks for the ping here and helping address all these issues!
